src/org/opensolaris/opengrok/configuration/messages/ProjectMessage.java

```diff
@@ -102,13 +115,25 @@ private void validateMore(RuntimeEnvironment env) throws Exception {
             env.getIgnoredNames()));
     }
 
+    private String getCommand() {
+        String command = getText();
+        
+        // get/set need special treatment since the text is overloaded with actual
+        // command and its contents.
+        if (command.startsWith("set ") || command.startsWith("get ")) {
+            command = getText().substring(0, 3);
+        }
+        
+        return command;
+    }
+    
     @Override
     protected byte[] applyMessage(RuntimeEnvironment env) throws Exception {
-        String command = getText();
+        String command = getCommand();
 
         validateMore(env);
-
-        switch (command) {
+        
+        switch (getCommand()) {
```

Comment:
Why call `getCommand()` for the second time? The result is already in `command` variable.